math/src/polismath/math/conversation.clj

```diff
@@ -18,13 +18,8 @@
     [plumbing.core :as plmb]
     [plumbing.graph :as graph]
     [bigml.sampling.simple :as sampling]
-    ;[alex-and-georges.debug-repl :as dbr]
     [taoensso.timbre :as log]
-    [clojure.spec.gen.alpha :as gen]
-    [clojure.test.check.generators :as generators]
-    [tech.v3.dataset :as ds]
-    [tech.v3.datatype :as dt]))
```

Comment:
Fine to remove these for now, though it's on our roadmap to move to the `tech.*` stack for array and dataframe-type datastructures.